symengine/subs.h

```diff
@@ -144,6 +144,25 @@ class SubsVisitor : public BaseVisitor<SubsVisitor>
         result_ = x.create(v);
     }
 
+    void bvisit(const Contains &x)
+    {
+        RCP<const Basic> a = apply(x.get_expr());
+        RCP<const Set> b = rcp_static_cast<const Set>(apply(x.get_set()));
```

Comment:
`x.get_set()` always returns the object of type `Set`, then doesn't that guarantee that output is a `Set`?